stylecop.json

```diff
@@ -0,0 +1,5 @@
+{
+    "$schema": "https://raw.githubusercontent.com/DotNetAnalyzers/StyleCopAnalyzers/master/StyleCop.Analyzers/StyleCop.Analyzers/Settings/stylecop.schema.json",
+    "settings": {
+    }
+}
```

Comment:
Empty `stylecop.json` that we might want to populate later. It's used to pass data into the analyser, such as the name of the project for documentation/header checking.